index.html

```diff
@@ -308,14 +316,18 @@ <h3 class="skill-title">Skills</h3>
         </li>
       </ul>
     </section>
-    <footer>
-      <div class="contact-form-wrapper">
+
+    <footer id="contact">
+      <div>
         <form class="contact-form" method="POST" action="https://formspree.io/f/xpzbadnj">
+          <div class="contact-form-wrapper">
           <div class="contact-info">
             <p class="footer-text">
               I'm always interested in hearing about new projects, so if you'd
               like to chat please get in touch.
             </p>
```

Comment:
- [ ] To correctly match the Figma template design, I think it would be better if you use the correct color like the one one used on Figma to the above text (i.e. `#262626`)